fdbclient/NativeAPI.actor.cpp

```diff
@@ -7296,6 +7304,10 @@ void Transaction::setOption(FDBTransactionOptions::Option option, Optional<Strin
 		trState->options.enableReplicaConsistencyCheck = true;
 		break;
 
+	case FDBTransactionOptions::CONSISTENCY_CHECK_REQUIRED_REPLICAS:
+		validateOptionValuePresent(value);
+		trState->options.requiredReplicas = extractIntOption(value, -2, std::numeric_limits<int64_t>::max());
```

Comment:
In the context of the comment https://github.com/apple/foundationdb/pull/11348/commits/d6f6b4512550988b76c6bc95339fb864e4b020ef#r1585855408 above: this is the call that would cause a throw if use "Int-type" for "requiredReplicas".